src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -0,0 +1,1125 @@
+
+Our goal is to provide you the same world-class observability experience whether your data comes from New Relic agents or OpenTelemetry. The simple truth is that some experiences will be different because OpenTelemetry is a newer technology than New Relic agents, and it has a different data model. Though OpenTelemetry is rapidly growing in scope and maturity, there are still capabilities of the New Relic platform that aren’t yet supported by OpenTelemetry instrumentation or specifications. We are committed to adding more and more support for OpenTelemetry specifications in the New Relic platform.
+
+We recommend you start with the table at the beginning that shows high-level comparisons. Then, you can use that information to focus on the detailed sections that are most important to you. The sections that follow have icons that describe whether you can expect particular experiences in New Relic:
+
+* ✅ Supported in New Relic
+* 🟡 Limited support in New Relic
+* ❌ Not supported in New Relic
+
+<Callout variant="tip">
+  Here are some ways navigate through these comparisons:
+  * Scan the headings in the right navigation pane.
+  * Use `CONTROL-F` to search the page for key phrases.
+  * Expand all collapser sections by pressing `s` or hide with `h`.
+</Callout>
+
+## High-level comparisons [#overview]
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Feature
+      </th>
+      <th>
+        New Relic agents
+      </th>
+      <th>
+        OpenTelemetry in New Relic
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        Collect metrics, traces, and logs
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Collect custom events
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        ❌ Not specified with OpenTelemetry
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Measure and track SLIs/SLOs
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 Spans only at present
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Optimize application performance
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 OpenTelemetry instrumentation varies significantly by language and is improving. New Relic agents have overall broader coverage of languages.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Discover root causes in complex systems
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 Front-end and Kubernetes observability is limited
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Triage and fix errors and exceptions
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 Front-end and Kubernetes observability is limited
```

Comment:
"... is limited **unless combined with New Relic agents**"

Reply:
@rhetoric101 This suggestion is slightly different from the one above on "Discover root causes..." since k8s doesn't really apply to the "Triage and fix errors..." use.